scripts/noneq_adaptation/utils.py

```diff
@@ -0,0 +1,27 @@
+from pymbar import EXP
+import numpy as np
+from typing import Callable, List
+from md.states import CoordsVelBox
+
+Lambda = float
+Energies = np.array
+VectorizedPotentialEnergy = Callable[[List[CoordsVelBox], Lambda], Energies]
+Samples = List[CoordsVelBox]
+
+
+def compute_work_increments(
```

Comment:
Do we want to have imports between files in the scripts dir? The dir name would imply to me that everything is 'stand alone'.

Reply:
What would you prefer? (Intent of [splitting](https://github.com/proteneer/timemachine/pull/442/commits/39e2eebe27fa48e005bf01bfdfafe0d7f759bdca) into 3 scripts rather than 1 was to ease later extraction to wherever in the `docking` or `fe` modules would be most appropriate, if we decide to use this style of schedule adaptation)

Reply:
It would see that if the logic is used by three scripts and intended to be reused, should live in a meaningful timemachine module.

Reply:
Since this seems more applicable to docking, perhaps that is where it should go? Or having an `optimize.protocol` module.

This seems within scope, just don't want to 'lose' production code to a scripts directory

Reply:
Sounds good! Adding to the `optimize` module makes more sense than the alternatives I had initially thought of.